src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterOptions.cs

```diff
@@ -30,11 +31,27 @@ public sealed class ZipkinExporterOptions
         internal const int DefaultMaxPayloadSizeInBytes = 4096;
 #endif
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ZipkinExporterOptions"/> class.
+        /// Initializes zipkin endpoint.
+        /// </summary>
+        public ZipkinExporterOptions()
+        {
+            try
+            {
+                this.Endpoint = new Uri(Environment.GetEnvironmentVariable("OTEL_EXPORTER_ZIPKIN_ENDPOINT") ?? "http://localhost:9411/api/v2/spans");
+            }
+            catch (Exception ex)
+            {
+                ZipkinExporterEventSource.Log.FailedEndpointInitialization(ex);
```

Comment:
The exception is being caught but Endpoint is being left as null. Per https://github.com/open-telemetry/opentelemetry-dotnet/pull/2123#discussion_r666357389 it should log the error and fallback to default.

Reply:
Ah github :) I see the updates after I post ...